data/src/main/java/com/github/leuludyha/data/repository/LibraryRepositoryImpl.kt

```diff
@@ -44,17 +44,14 @@ class LibraryRepositoryImpl(
     override fun getAuthorRemotely(authorId: String): Flow<Result<Author>> =
         libraryRemoteDataSource.getAuthor(authorId)
 
-    override suspend fun saveWorkLocally(work: Work) {
-        TODO("Not yet implemented")
```

Comment:
Does invoking a function with a `TODO(args)` throws an exception ? Or what does it do ?

Reply:
It does throw an exception.